src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,127 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+
+import duke.components.DukeException;
+import duke.components.Storage;
+import duke.components.TaskList;
+import duke.components.Ui;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+/**
+ * Class of commands that adds to the existing list.
+ */
+public class AddCommand extends Command {
+    /**
+     * type contains the type of the command.
+     *
+     * task contains the name of the Task.
+     *
+     * deadline contains the by of the Task.
+     *
+     * from contains the start of the Task.
+     *
+     * to contains the end of the Task.
+     *
+     * isExit contains whether to terminate the bot.
+     */
+    private String type;
+    private String task;
+    private LocalDateTime deadline;
+    private LocalDateTime from;
+    private LocalDateTime to;
+    private boolean isExit = false;
```

Comment:
Good job in using the proper naming convention for boolean variables, and in other places as well.